kaska/utils.py

```diff
@@ -175,10 +177,15 @@ def save_output_parameters(time_grid, observations, output_folder, parameter_nam
     for (param, data) in zip(parameter_names, output_data):
         for band, tstep in enumerate(time_grid):
             this_date = tstep.strftime("%Y%j")
-            if chunk is None:
-                outfile = output_folder/f"s2_{param:s}_A{this_date:s}.tif"
-            else:
-                outfile = output_folder/f"s2_{param:s}_A{this_date:s}_{chunk:s}.tif"
+            
+            # Compose the output file name from pattern (satellite),
+            # parameters, dates and, optionally, chunk
+            extra = ""
+            if chunk is not None:
```

Comment:
```suggestion
            if chunk:
```
`None` evaluates as False, so unless we want to allow `chunk = ''` (empty string) to get an extra `_` in `outfile`, then my suggestion is invalid.

Reply:
Done